stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1654,6 +1654,17 @@ class Stripe internal constructor(
     }
 
     companion object {
+        /**
+         * If the SDK receives a "Too Many Requests" (429) status code from Stripe,
+         * it will automatically retry the request using exponential backoff.
+         *
+         * The default value is 3.
+         *
+         * See https://stripe.com/docs/rate-limits for more information.
+         */
+        @JvmField
+        var MAX_RETRIES: Int = 3
```

Comment:
> I think upper case are supposed to be immutable constants but this value is mutable, since ApiRequestExecutor.maxRetries pulls this value directly, changing the value prior to initializing Stripe object would impact API behavior.

👍 

> If this is a field that's supposed to alter API behavior, we probably should make it as a val parameter of Stripe constructor with default value to val MAX_RETRIES, and let ApiRequestExecutor.maxRetries pull the val instead?

It's a little bit more complicated, because we sometimes make requests on behalf of the user (i.e. they don't create the `Stripe` instance themselves). We've historically put some of these global configs here, so I think it makes sense to continue doing that for now.

Reply:
oh if that's the case probably at least make it it internal? we can expose it when we have the proper API to let user change it

Reply:
I suppose we can defer making it configurable for another PR. I'll do that.